icu4c/source/i18n/unicode/measunit.h

```diff
@@ -13,6 +13,7 @@
 #ifndef __MEASUREUNIT_H__
 #define __MEASUREUNIT_H__
 
+#include <utility>
```

Comment:
Sorry, that didn't work...
Some of our integration tests try to compile each header file on its own with a C compiler, and that chokes on this C++ header :-(
Could you please move this include to just before the unicode/unistr.h include?

Alternatively, you could leave it here but make it conditional:
```
#ifdef __cplusplus
#include <utility>
#endif
```